### PR TITLE
Set bug or feature label depending on issue template

### PR DESCRIPTION
**Pull request by galexrt**
**Description of your changes:**
Sets `bug` or `feature` label depending on the issue template chosen.

**Checklist:**
- [ ] Documentation has been updated, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make codegen`) has been run to update object specifications, if necessary.
- [ ] Comments have been added or updated based on the standards set in [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md#comments)

[skip ci]

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 name: Feature request
 about: Suggest an idea/feature for this project
+labels: feature
```
Comment (jbw976) [inline .github/ISSUE_TEMPLATE/feature_request.md]: we don't have a feature label defined: https://github.com/rook/rook/labels.  What do you think about `enhancement`? https://github.com/rook/rook/labels/enhancement

Comment (travisn) [reply .github/ISSUE_TEMPLATE/feature_request.md]: What if we rename `enancement` to `feature`?

Comment (jbw976) [reply .github/ISSUE_TEMPLATE/feature_request.md]: That's fine with me :)

Comment (galexrt) [reply .github/ISSUE_TEMPLATE/feature_request.md]: As there don't seem to be objections against renaming the label, I went ahead and renamed it.

**Comment by galexrt** [comment]
@jbw976 @travisn GitHub label renamed, PTAL.

**Pull request merged**